graphql-server/metadata/tables.yaml

```diff
@@ -305,8 +305,10 @@
     permission:
       check:
         turn:
-          player_id:
-            _eq: X-Hasura-User-Id
+          game:
+            players:
+              id:
+                _eq: X-Hasura-User-Id
```

Comment:
relax the permission, host (and other players in the future) can skip players/teams.